xchange-upbit/src/main/java/org/knowm/xchange/upbit/dto/marketdata/UpbitCandleStickData.java

```diff
@@ -0,0 +1,120 @@
+package org.knowm.xchange.upbit.dto.marketdata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+
+public class UpbitCandleStickData {
```

Comment:
you can use lombok for boilerplate-code here

Reply:
Thanks for the suggestion! I've used Lombok to simplify the code.